Tests/Command/GenerateCommandTest.php

```diff
@@ -18,6 +18,13 @@
 
 abstract class GenerateCommandTest extends \PHPUnit_Framework_TestCase
 {
+    protected $kernelRootDir;
+
+    public function setup()
+    {
+        $this->kernelRootDir = sys_get_temp_dir();
```

Comment:
I would generate a unique directory name and also clean it up after the tests have been executed.